webpack/react_app/components/HostKebab/KebabItems.js

```diff
@@ -13,7 +13,11 @@ const HostKebabItems = () => {
 
   if (!consoleUrl) return null;
   return (
-    <DropdownItem icon={<CodeIcon />} href={consoleUrl}>
+    <DropdownItem
```

Comment:
Can't we just add `target="_blank"`? It should get passed to whatever the underlying element it

Reply:
I agree with this, its more recommended to use
``` js
 <DropdownItem
      icon={<CodeIcon />}
      href={consoleUrl}
      target="_blank"
      rel="noreferrer"
    >
 ```